config/bpfd/bpfd.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
```

Comment:
We should probably add some comments with an explanation of what's going on with this `DaemonSet`, right?